AssignedTo.h

```diff
@@ -0,0 +1,22 @@
+//
+//  AssignedTo.h
+//  FilterSearchAttempt
+//
+//  Created by Lizzy Randall on 5/5/15.
+//  Copyright (c) 2015 Lizzy Randall. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface AssignedTo : NSObject {
+    NSString *name;
```

Comment:
the objective c compiler now automatically adds these ivars for you, it also automatically adds the @synthesize statements for you. So now if you want a property you can just add a @property statement